### ISSUE TITLE
NVDA fails to read book contents with Kindle for PC

### ISSUE THREAD

**Issue by PratikP1** [nvaccess/nvda#7661]
With the latest version of Kindle for PC, NVDA fails to read the book contents for any book in the library. I updated Kindle for PC from 1.19X to 1.2X. The problem occurs in the previous version as well as the latest version I installed. I have attached object log as well as the info log. The problem occurs with or without add-ons disabled. The results are the same regardless of whether step 1 is followed.

### Steps to reproduce:
1. Restart NVDA without add-ons
2. start Kindle for PC.
3. Open a book (if a previous book hasn't been opened already).
4. Try to read the book with the browse mode.
5. Go to The table of contents and move to other sections of the book and read.
6. Open other books and repeat steps.

### Expected behavior:
NVDA should read the contents of the book by using normal navigation commands.

### Actual behavior:
None of the contents is read. Often, when the book is opened, Kindle's automatic TTS launches. It has to be manually turned off.

### System configuration:
NVDA version:
NVDA version next-14511,339707cd

NVDA Installed or portable:
Installed

Other information:
> Example: Running in a VM

Windows version:
Windows 10 Pro Insider 1709 Build 10.0.17017 

Name and version of other software in use when reproducing the issue:
Kindle version 1.20.1.47037

### Other questions:

Does the issue still occur after restarting your PC?
Yes

Have you tried any other versions of NVDA?
2017-3

[NVDA Kindle Book page view object.txt](https://github.com/nvaccess/nvda/files/1385200/NVDA.Kindle.Book.page.view.object.txt)
[NVDA Kindle Not speaking.txt](https://github.com/nvaccess/nvda/files/1385201/NVDA.Kindle.Not.speaking.txt)

**Comment by feerrenrut** [comment]
I was not able to reproduce this problem, to test I used the free (and screen reader supported) book: [A tale of two cities](http://a.co/9BLShWi)
With Kindle for PC version `1.20.1.47037` and NVDA version `next-14535,3aad54e0`

It's important to note that under the "Product details" heading for the book, the "Screen Reader" support is listed. Not all books have screen reader support. Please can you confirm that the book you are trying to read has screen reader support?

Please also try with the free ebook listed above.

**Comment by PratikP1** [comment]
@feerrenrut  wrote:
< Not all books have screen reader support. Please can you confirm that the book you are trying to read has screen reader support? Please also try with the free ebook listed above.

I have tested with the ebook you have listed above. I am getting the same results. I am aware of the fact that not all Kindle books support TTS/screen reader functionality. All books in my collection have screen reader capability. I have tested this behavior with multiple books including public domain and proprietary books.

**Comment by feerrenrut** [comment]
Quentin has also been unable to reproduce this issue using the same book as PraticP1, he is on the same insider build `version 1709 Build 10.0.17017` and using NVDA version `next-14541,267b5407`.

**Comment by PratikP1** [comment]
Thank you for doublechecking. I uninstalled Kindle for PC, cleared all files in the library, and reinstalled. Now it works as expected with NVDA.

**Comment by ehollig** [comment]
Closing as worksforme, from above comment.
